### DISCUSSION TITLE
Using json instead of js for translation files.

### DISCUSSION THREAD

**Discussion by fwiwDev** [intlify/bundle-tools#106] [Q&A]
Hi,

I am using the Quasar boilerplate to get started with an app. It compiles well but I cannot figure out how to use json files in `src\i18n` instead of js files. Even adding a json file to the directory causes this error:

`[plugin:vite-plugin-vue-i18n] Cannot read properties of undefined (reading 'message')`

I have this set
`
      vitePlugins: [
        ['@intlify/vite-plugin-vue-i18n', {
          // if you want to use Vue I18n Legacy API, you need to set 'compositionOnly: false'
          // compositionOnly: false,

          // you need to set i18n resource including paths !
          include: path.resolve(__dirname, './src/i18n/**')
        }]
      ]
`

Has anyone run into this and are you able to offer any solutions please?

Thank you!

**Comment by BernhardSchlegel** [comment]
Any updates on this?
